.github/workflows/storybook.yml

```diff
@@ -0,0 +1,25 @@
+name: Deploy Storybook to GitHub Pages
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Exit if not on master branch
+        if: endsWith(github.ref, 'master') == false
+        run: exit 0
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org
+      - name: Install deps and deploy to GitHub Pages
+        run: |
+          yarn
+          yarn docs:deploy -- --ci
+        env:
+          GH_TOKEN: ${{ github.actor }}:${{ secrets.GITHUB_TOKEN }}
```

Comment:
This isn't new - bits from previous PR.  Not sure why it's showing up in this 🤷‍♂️